app/src/main/java/com/it3048/mobiledeviceproject/previous_button_fragment.kt

```diff
@@ -18,13 +18,15 @@ class SecondFragment : Fragment() {
             savedInstanceState: Bundle?
     ): View? {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_second, container, false)
+        return inflater.inflate(R.layout.previous_button_fragment.xml, container, false)
     }
+// line 21: fragment_second.xml -> a little confusing as to what this fragment is.
+//the xml file has a button labeled "Previous" on it. I rename the name of this fragment, and the xml file to "previous_button_fragment.
```

Comment:
Putting conversation/discussion  in code comments is bad practice, as that clutters up the source code and does not belong in source code files.  I mentioned that in the code review instructions.    Discussion and comments are very good, but have that in the appropriate place, which is in the "Files Changed" tab in the pull request in GitHub (which is where I am writing this and you are reading this).  That way, you can have a conversation with replies, right where the code is happening, and without cluttering up the source code itself.